graylog2-web-interface/src/components/configurations/UrlWhiteListForm.jsx

```diff
@@ -0,0 +1,219 @@
+        }
+        break;
+      default:
+        break;
+    }
+  };
+
+  const _onInputChange = (event: SyntheticInputEvent<EventTarget>, idx: number, type: string) => {
+    if (type === regex) {
+      const debouncedValidate = debounce(_validate, 500);
+      debouncedValidate(event.target.name, idx, type, FormUtils.getValueFromInput(event.target));
+    }
+    _validate(event.target.name, idx, type, FormUtils.getValueFromInput(event.target));
+  };
+
+  const _onUpdateType = (idx: number, type: string) => {
+    const stateUpdate = cloneDeep(config);
+    _validate('value', idx, type, stateUpdate.entries[idx].value);
+  };
+
+  const _getErrorMessage = (type: string) => {
+    return type === regex ? 'Not a valid Java regular expression' : 'Not a valid URL';
+  };
+
+  const _getSummary = () => {
+    return (config.entries.map((url, idx) => {
+      return (
+      // eslint-disable-next-line react/no-array-index-key
+        <tr key={url.id}>
+          <td style={{ verticalAlign: 'middle', textAlign: 'center' }}>{idx + 1}</td>
+          <td>
+            <Input type="text"
+                   id={`title-input${idx}`}
+                   ref={(elem) => { inputs[`title${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'Required field' : null}
+                   name="title"
+                   bsStyle={validationState.errors[idx] && validationState.errors[idx].title && !validationState.errors[idx].title.valid ? 'error' : null}
+                   onChange={event => _onInputChange(event, idx, url.type)}
+                   defaultValue={url.title}
+                   required />
+          </td>
+          <td>
+            <Input type="text"
+                   id={`value-input${idx}`}
+                   ref={(elem) => { inputs[`value${idx}`] = elem; }}
+                   help={validationState.errors[idx] && validationState.errors[idx].value && !validationState.errors[idx].value.valid ? _getErrorMessage(url.type) : null}
+                   name="value"
+                   bsStyle={validationState.errors[idx] && validationState.errors[idx].value && !validationState.errors[idx].value.valid ? 'error' : null}
+                   onChange={event => _onInputChange(event, idx, url.type)}
+                   defaultValue={url.value}
+                   required />
+          </td>
+          <td>
+            <Input id={`url-input-type-${idx}`}
+                   required
+                   autoFocus>
+              <Select clearable={false}
+                      options={options}
+                      matchProp="label"
+                      onChange={option => _onUpdateType(idx, option)}
+                      value={url.type} />
+            </Input>
+          </td>
+          <td>
+            <Button onClick={event => _onRemove(event, idx)}>
+              <Icon name="fa-trash" style={{ cursor: 'pointer' }} />
```

Comment:
This inner style shouldn't be necessary now.

Reply:
removed